lib/takeaway.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'menu'
+require_relative 'twilio'
+
+class Takeaway
+
+  attr_reader :menu, :new_order_list, :total_order
+
+  def initialize(menu = Menu.new)
+    @menu = menu
+    @new_order_list = Array.new
+    @total_cost = 0
+  end
+
+  def show_menu
+    @menu
+  end
+
```

Comment:
comment about your 'show_menu' method^:

I put this method in my menu class. I'm not sure if this is necessarily better, but I felt the functionality of 'displaying the menu' was more in line with a menu class with a hardcoded menu, than a takeaway/order class. As it stands, you're menu class doesn't really have much functionality beyond displaying the menu (i.e. there are no other methods/you can't change the menu there) so you might as well have just inserted it as a constant in your takeaway class. By adding the 'display' menu method to it, it seems to have more of a function